HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_01.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task1(log)
+    rows = log.split("\n")
+    rows.select { |str| str.include?('error') }[0].to_s
```

Comment:
```suggestion
    log.split("\n").select { |str| str.include?('error') }[0].to_s
```

and use `find` instead of `select` ( reed why and change your code ^_^ )